### ISSUE TITLE
Kontoname + Vorname unnötig lang/breit

### ISSUE THREAD

**Issue by Weissrolf** [fsinfuhh/Bitpoll#118]
Hallo.

Wenn ich mit Konto eine Umfrage erstelle, dann wird mein Name immer als Kontoname (Vorname) eingetragen. Dadurch wird das Namensfeld links unnötig breit, was vor allem auf Smartphones zu unnötigem Scrollen führt.

Leider scheint es nirgends möglich zu sein, dass zu ändern?!

**Comment by Akasch** [comment]
Nein, bisher geht das nicht. Ich habe die Überlegung einen Anzeigenamen welcher editierbar ist einzuführen.
